app/stories/components/ClinicSelectionList.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from '@storybook/client-api'
```

Comment:
Notes for reviewers: This story is a little bonkers because it has so many states passed in as props.

**Question:** Is there a better way to do this?

Reply:
Would this be better if the story was less interactive and we broke out different states for each of these potential options?